CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.12.2 (2020-12-01)
+
+Version 2.11.7 fixes several issues to improve the reliability of the cluster functionality.
```

Comment:
Are you sure about this?

Reply:
Nope, too much copy paste there :see_no_evil: